<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>1131</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="del_inventory_btn">
      <property name="text">
       <string>Удалить из инвентаря</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_inventory_btn">
      <property name="text">
       <string>Добавить в инвентарь</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>781</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="show_inventory_btn">
      <property name="text">
       <string>Весь список</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="show_on_hands_btn">
      <property name="text">
       <string>На руках</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="show_items_in_storage_btn">
      <property name="text">
       <string>На складе</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="list_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Инвентарь на складе</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>560</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>381</width>
     <height>531</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Вернул</string>
    </attribute>
    <widget class="QPushButton" name="in_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Вернул</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Инв.Номер:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Наименовение:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сдающий:</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="search_in_table_in_btn">
        <property name="text">
         <string>Найти в Таблице</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="take_in_table_in_btn">
        <property name="text">
         <string>Взять из Таблицы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="inventory_in">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_item_in">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_owner_in">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Взял</string>
    </attribute>
    <widget class="QPushButton" name="out_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Взял</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сотрудник:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="owner_out">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Инв.Номер:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Наименование:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inventory_number_out">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_out">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="search_in_table_out_btn">
        <property name="text">
         <string>Найти в Таблице</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="take_in_table_out_btn">
        <property name="text">
         <string>Взять из Таблицы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Инвентарный номер</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Наименование</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Владелец</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="search_text">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="help_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>781</width>
     <height>451</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>inventory_num</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>owner</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>extra</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="item_edit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>search_btn</tabstop>
  <tabstop>show_inventory_btn</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>del_inventory_btn</tabstop>
  <tabstop>add_inventory_btn</tabstop>
  <tabstop>show_on_hands_btn</tabstop>
  <tabstop>show_items_in_storage_btn</tabstop>
  <tabstop>in_btn</tabstop>
  <tabstop>out_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
